install.sh

```diff
@@ -14,13 +15,38 @@ alpine_install() {
     pip3 install docker-compose
     # install noma
     python3 setup.py develop
+}
+
+run_noma() {
     # run noma
     noma --version || exit 1
     noma --help || exit 1
     run_tests || exit 1
     start_noma
 }
 
+debian_install() {
+    if [ -x "$(command -v apt-get)" ]; then
+        # docker
+        curl -fsSL https://get.docker.com -o get-docker.sh
+        sh get-docker.sh
+        # dependencies
+        apt-get install python3-pip python3-cffi libffi-dev libssl-dev git -y
```

Comment:
I usually like the `-y` as close to `install` as possible, but that's a nit.

Reply:
Agree for readability